operator/internal/manifests/openshift/configure.go

```diff
@@ -194,14 +195,14 @@ func ConfigureRulerStatefulSet(
 }
 
 // ConfigureOptions applies default configuration for the use of the cluster monitoring alertmanager.
-func ConfigureOptions(configOpt *config.Options) error {
+func ConfigureOptions(configOpt *config.Options, uwam bool, token, caPath, monitorServerName string) error {
```

Comment:
Is it possible to enclose the newly added parameters into `config.Options`?

Reply:
`config.Options` is what we are trying to configure here